src/api/payment-requests/payment-requests.md

```diff
@@ -105,13 +105,17 @@ version (documented on this page) and the "legacy" version (documented at
 Once you have made that payment, you can use the transaction id to [Pay a Payment Request](#pay) using the legacy payment API.
 
 <a name="payment-condition">
+
 ### Payment Condition
-The `conditionsEnabled` property should be true when creating the Payment Request to opt into Payment Conditions.
-If Payment Conditions are not opted into, some asset types may not be available as a [Payment Option](#payment-option).
 
-Conditions can either be [accepted](#accept-payment-condition) or declined.
-Conditions must be accepted for a Payment Request to be `paid` with asset types that require those conditions.
-Possible payment conditions include, among others, confirming proof of ID or confirming a promotional item was purchased.
+Some asset types require conditional approval to pay. Possible payment conditions include confirming proof of ID
+or confirming a promotional item was purchased.
+
+The `conditionsEnabled` flag should be set to true when [Creating a Payment Request](#create-a-payment-request) or these asset
+types may not be available as a [Payment Option](#payment-option).
+
+Conditions can either be [accepted](#accept-payment-condition) or [declined](#accept-payment-condition).
+Conditions must be accepted for a Payment Request to be `paid` with an asset types that requires conditional approval.
```

Comment:
Can you add that if payment conditions are declined then the payment will be canceled?

Reply:
Done ✅